src/pages/assetGroup/AssetGroup.jsx

```diff
@@ -19,15 +19,37 @@ import Link from '../../components/Link';
 import MoreMenu from '../../components/MoreMenu';
 import ConfirmDelete from '../../components/ConfirmDelete';
 import EntityHeader from '../../components/EntityHeader';
+import CustomAlert from '../../components/Alert';
 import AGSTable from './AGSTable';
 
+const POLLING_INTERVAL = 5000; // 5 seconds
+
+function deriveRefetchInterval(resultData, query) {
+  const { complete, status } = {
+    ...resultData?.data?.progress_preparation,
+  };
```

Comment:
why not just 
```
const { complete, status } = resultData?.data?.progress_preparation

or if that doesn't work 

const { complete, status } = resultData?.data?.progress_preparation || {}
```

Reply:
You're right. That's legacy from when I wasn't defining primitives.

Reply:
Fixed in https://github.com/WildMeOrg/codex-frontend/pull/366/commits/49f3271d91e3f4cd39c39759146eaa482555bb08